src/platform/icelake/include/platform/alh.h

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
+
+#ifndef __PLATFORM_ALH_H__
```

Comment:
Add #ifdef __SOF_ALH_H__ to force inclusion only from sof/alh.h. See other platform headers.

Reply:
I decided to throw an error if included in a wrong place instead, that way the users will not get the silent treatment.